packages/lodestar/src/network/interface.ts

```diff
@@ -33,9 +33,17 @@ export interface IReqResp {
   unregisterHandler(): ReqRespHandler | null;
 }
 
+export enum NetworkEvent {
+  peerConnect = "peer:connect",
+  peerDisconnect = "peer:disconnect",
+  gossipStart = "gossip:start",
+  gossipStop = "gossip:stop",
+  gossipHeartbeat = "gossipsub:heartbeat",
```

Comment:
I thought we had convention to UPPER_CASE enums?

Reply:
I just followed the convention used in `ChainEvent`:
https://github.com/ChainSafe/lodestar/blob/49df11a86a3003a80b97ccca477eaae5a90765df/packages/lodestar/src/chain/emitter.ts#L19